enhancements/microshift/microshift-updateability-ostree.md

```diff
@@ -105,34 +106,33 @@ Design aims to implement following principles:
 ### Backing up and restoring MicroShift data
 
 MicroShift needs to fit into workflow of ostree-based systems: device can be
-upgraded by staging and booting new ostree commit, and rolled back if it's
+upgraded by staging and booting new ostree deployment, and rolled back if it's
 unhealthy or admin wishes to do it.
 
 To achieve that MicroShift needs to make backups and be able to restore them in
 sync with ostree lifecycle.
 Backing up and restoring will happen on boot, just before MicroShift cluster
 runs (for alternative see [Performing backup on shutdown](#performing-backup-on-shutdown))
-and with MicroShift cluster (i.e. etcd, kube-apiserver, etc.) not being off.
+and with MicroShift cluster (i.e. etcd, kube-apiserver, etc.) not running.
 As a general rule: if previous boot was healthy, data will be backed up, if boot
 was unhealthy, data will be restored.
 If system rolled back, we want compatible and healthy data to be restored,
-but, if previous commit was healthy, after backing up its data.
+but, if previous deployment was healthy, after backing up its data.
 
 To provide ability to rollback and restore MicroShift data, backup for each
-ostree commit/deployment will be kept.
+ostree deployment will be kept.
```

Comment:
Do we want to specify that the data will be kept on device?

Reply:
Done